src/main/java/seedu/financialplanner/commands/HelpCommand.java

```diff
@@ -0,0 +1,159 @@
+package seedu.financialplanner.commands;
+
+import seedu.financialplanner.utils.Ui;
+
+import java.util.ArrayList;
+
+import static seedu.financialplanner.utils.Parser.ADD_CASHFLOW_COMMAND;
+import static seedu.financialplanner.utils.Parser.ADD_REMINDER_COMMAND;
+import static seedu.financialplanner.utils.Parser.ADD_STOCK_COMMAND;
+import static seedu.financialplanner.utils.Parser.BALANCE_COMMAND;
+import static seedu.financialplanner.utils.Parser.BUDGET_COMMAND;
+import static seedu.financialplanner.utils.Parser.DELETE_CASHFLOW_COMMAND;
+import static seedu.financialplanner.utils.Parser.DELETE_STOCK_COMMAND;
+import static seedu.financialplanner.utils.Parser.EXIT_COMMAND;
+import static seedu.financialplanner.utils.Parser.FIND_COMMAND;
+import static seedu.financialplanner.utils.Parser.HELP_COMMAND;
+import static seedu.financialplanner.utils.Parser.LIST_COMMAND;
+import static seedu.financialplanner.utils.Parser.OVERVIEW_COMMAND;
+import static seedu.financialplanner.utils.Parser.SET_GOAL_COMMAND;
+import static seedu.financialplanner.utils.Parser.VISUALIZATION_COMMAND;
+import static seedu.financialplanner.utils.Parser.WATCHLIST_COMMAND;
+
+public class HelpCommand extends Command {
```

Comment:
would it be better to make helpcommand a singleton?

Reply:
True, can save resource waste due to instantiation